src/App.jsx

```diff
@@ -1,5 +1,36 @@
 import 'bulma/css/bulma.css';
+import cn from 'classnames';
+
 import './App.scss';
```

Comment:
first there should be external imports, then internal
so `import './App.scss';` should goes after other it this list